spec/controllers/tasuku/taskables/questions/answers_controller_spec.rb

```diff
@@ -56,5 +57,49 @@ module Tasuku
       end
     end
 
+    describe "PUT 'update'" do
+      it 'routes' do
+        expect(put: '/taskables/questions/1/answers/1').to route_to(
+          action: 'update',
+          controller: 'tasuku/taskables/questions/answers',
+          question_id: '1',
+          id: '1'
+        )
+      end
+
+      context 'with a correct answer' do
+        let(:user)     { create :user }
+        let(:question) { create :question_with_options, correct_option: 1 }
+
+        let(:vote) { create :question_vote, option: question.options.first }
+        let(:question_answer) { create :question_answer, author: user, votes: [vote] }
+
+        let(:new_vote) { create :question_vote, option: question.options.last }
+        let(:params) { { question_id: question.id, id: question_answer.id, taskables_question_answer: { option_ids: [new_vote.id] }  } }
+
+        before { request.env['HTTP_REFERER'] = 'http://example.org' }
+        before { allow(subject).to receive(:current_user).and_return(user) }
+
+        #it_behaves_like 'redirectable' do
+        #  let(:action) { :update }
+        #  let(:verb)   { :put }
+        #  let(:update_answer) { true }
+        #end
+        # TODO: Need to solve this later
```

Comment:
I was not able at the time to add it conditional tests to this part so i commented it out for questions that can be updated. We might need to create a new it_behaves_like to test this behavior. maybe 'redirectable_update_answer'